app/models/conference.rb

```diff
@@ -93,6 +93,28 @@ def user_registered? user
   end
 
   ##
+  # Checks if domain correctly points to the hosted version
+  # This feature is enabled only if ENV['OSEM_HOSTNAME'] is present
+  #
+  # ====Returns
+  # * +true+ -> If the custom domain has a CNAME record for the hosted version
+  # * +false+ -> If the custom domain does not have a CNAME record for the hosted version
+  def check_custom_domain
+    require 'resolv'
+
+    unless ENV['OSEM_HOSTNAME'].nil?
+      cname_record = Resolv::DNS.new.getresources(custom_domain, Resolv::DNS::Resource::IN::CNAME)
+      if cname_record.present?
+        return ENV['OSEM_HOSTNAME'] == Resolv::DNS.new.getresources(custom_domain, Resolv::DNS::Resource::IN::CNAME).first.name.to_s
+      else
+        return false
+      end
+    end
+
+    '--feature disabled--'
```

Comment:
There can be a situation in which `ENV['OSEM_HOSTNAME']` is not present. In which case it won't be possible to check if the custom domain has the correct CNAME. So, this feature will be disabled until the site admin sets the OSEM_HOSTNAME environment variable.
  
We can add the above information in the documentation about site admins about how to enable the feature.